torch_xla/csrc/tensor.cpp

```diff
@@ -226,8 +226,13 @@ xla::ComputationClient::DataPtr GetDeviceData(const at::Tensor& tensor,
 xla::ComputationClient::DataPtr GetDeviceData(at::Scalar value,
                                               at::ScalarType scalar_type,
                                               const Device& device) {
-  return GetDeviceData(at::scalar_tensor(value, at::TensorOptions(scalar_type)),
-                       device);
+  // Workaround since at::scalar_tensor doesn't support bfloat16 yet.
```

Comment:
This should be a utility function in torch_util.cpp/h, and a bug open with a reminder to remove the hack once scalar_tensor supports bfloat16.
Actually, it may be argued that this should be in pytorch main directly.